app/views/reminder_mailings/_reminder_mailing.html.slim

```diff
@@ -0,0 +1,12 @@
+tr
+  td= reminder_mailing.t_enum(:kind)
+  td= link_to reminder_mailing.creator.to_label, reminder_mailing.creator
+  td
+    - if reminder_mailing.sending_triggered
+      'Übermittelt
+    - else
+      'Nicht versandt
+  td= l(reminder_mailing.created_at.to_date)
+  td
+    = link_to 'Anzeigen', reminder_mailing, class: 'index-action-link'
+    = link_to 'Bearbeiten', edit_reminder_mailing_path(reminder_mailing), class: 'index-action-link'
```

Comment:
`- if !reminder_mailing.sending_triggered` 
should one be able to edit a mail which is anyway already sent out?